Firebase/android/app/src/main/res/layout/merge_login_view.xml

```diff
@@ -8,18 +8,19 @@
     android:layout_height="match_parent"
     android:orientation="vertical">
 
-    <FrameLayout
+    <com.novoda.bonfire.view.BubblyLinearLayout
```

Comment:
Why the switch from `FrameLayout` to `LinearLayout` with no changes to the contents? ⚠️